geonode/base/api/serializers.py

```diff
@@ -496,6 +521,18 @@ def to_internal_value(self, data):
         required=False,
     )
 
+    """
+     - Deferred / not Embedded --> ?include[]=executions
+    """
+    executions = DynamicRelationField(
```

Comment:
@afabiani this would throw an exception since `executions` is not a field on the ResourceBase. Also making it a property field always returns it in the response whether `deferred=True` is given or not.